lib/rules/widget-not-inline-matches.js

```diff
@@ -18,7 +18,8 @@ const matchesFns = [
 ];
 
 function isWidgetType(vNode) {
-  return getRoleType(vNode) === 'widget';
+  const roleType = getRoleType(vNode);
+  return roleType === 'widget' || roleType === 'composite';
```

Comment:
Undo this change since we'll leave `listbox` and `combobox` as widgets

Reply:
Undid this change 2d185f65